plotst/axis.typ

```diff
@@ -143,10 +145,21 @@
       pos.at(1) = -length + pos.at(1)
       invert_markings = -1
     }
-    
+    let arrow_size = 0pt
+    if axis.show_arrows {
+      // sets the size of arrows
+      arrow_size = 2pt
+    }
     if is_vertical(axis) {
       // Places the axis line
-      place(dx: pos.at(0), dy: pos.at(1), line(angle: -90deg, length: length, stroke: axis.stroke))
+      place(dx: pos.at(0), dy: pos.at(1), line(angle: -90deg, length: length - arrow_size * 2, stroke: axis.stroke))
+      
+      // draw the arrow at the end of the axis
+      if axis.show_arrows {
+        place(dx: pos.at(0), dy: pos.at(1) - length, 
+          polygon(fill: axis.stroke, (-arrow_size, arrow_size * 2), (0pt, -1pt), (arrow_size, arrow_size * 2))
```

Comment:
`fill` might break if you use something like `2pt + red`. But this can be a note in the docs if there is no good way to check.

Reply:
Thank you for the fast reply and the extensive testing, I don't see the error in my render. Can you send me a screenshot what you mean?

<img width="300" src="https://github.com/Pegacraft/typst-plotting/assets/6866008/2dd66571-0b50-45e2-8212-99f1e15a8c26">

Reply:
> give a programmer 7 lines of code and he'll find 20 errors, give him 2000 lines and he'll say it looks good

the `stroke` parameter in the axis allows true stroke inputs - `axis(…, stroke: 7pt + red, show_arrows: false)` is valid - however fill doesn't allow the same control.
Sadly you can't extract the color just yet. See typst/typst#790